scripts/get-npm-tag.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-commonjs */
+/* eslint-disable no-console */
+const { version } = require('../package.json');
+
+const getReleaseTag = _version => {
+  const [match = 'latest'] = _version.match(/[a-z]+/) || [];
+  return match;
+};
+
+// give this on to the main script
+console.log(getReleaseTag(version));
```

Comment:
There is maybe a more "cleaner" way to output a value?

Reply:
alternative is `process.stdout.write`. I'll change to that, only difference is that console.log will send a newline :

Reply:
I'm fine with this implementation then, `console.log` actually calls `process.stdout.write` internally.